lib/c0/c0sk.c

```diff
@@ -1397,11 +1397,12 @@ c0sk_cursor_update(struct c0_cursor *cur, u64 seqno, u32 *flags_out)
 HSE_USED HSE_COLD void
 c0sk_cursor_debug_base(struct c0sk *handle, u64 seqno, const void *prefix, int pfx_len, int skidx)
 {
+    bool eof;
+    merr_t err;
+    struct c0_cursor *cur;
     struct cursor_summary summary;
-    struct c0_cursor *    cur;
+    struct kvs_cursor_element elem;
```

Comment:
Please stop making gratuitous changes, really difficult to see what's relevant.  Maybe only if you're doing a major refactor...

Reply:
Hmmm, I guess I can revert it if you want. At some point, I just gave up.